stacks-signer/src/ping/mod.rs

```diff
@@ -0,0 +1,75 @@
+use serde_derive::{Deserialize, Serialize};
+mod processor;
+mod runloop;
+use crate::{
+    stacks_client::{PING_SLOT, SLOTS_PER_USER},
+    utils::ToSlotId,
+};
+use uuid::Uuid;
+
+pub use processor::RTTProcessor;
+pub use runloop::{Command, RunLoop};
+
+impl<'a> ToSlotId for &'a Packet {
+    fn slot_id(&self, signer_id: u32) -> u32 {
+        SLOTS_PER_USER * signer_id + PING_SLOT
+    }
+}
+
+/// What is written to the ping slot.
+#[derive(Serialize, Deserialize, Clone)]
+pub enum Packet {
+    /// Outgoing
+    Ping(Ping),
+    /// Incoming
+    Pong(Pong),
+}
+
+#[derive(Serialize, PartialEq, Clone, Deserialize, Debug)]
+/// A ping in a slot means someone has requested you to push a Pong into your slot.
+pub struct Ping {
+    id: u64,
+}
+#[derive(Serialize, Deserialize, Debug, Clone)]
+/// A pong in a slot means someone has responded to an RTT request.
+pub struct Pong {
+    id: u64,
```

Comment:
Same with the pong -- it should return the arbitrarily-large payload sent in the Ping.